rubrics/tool/src/main/frontend/webcomponents/sakai-rubric-grading-comment.js

```diff
@@ -41,7 +41,7 @@ export class SakaiRubricGradingComment extends RubricsElement {
         <div class="arrow"></div>
         <div class="popover-title" style="display: flex;">
           <div style="flex: auto;">
-            <label class="criterion-title" for="">
+            <label class="criterion-title">
```

Comment:
so is this not a label for anything? should just be a span or div?

Reply:
I thought that, but I just thought it best to leave it alone as it wasn't part of this ticket. I can dig in a bit more.